dwave/system/samplers/dwave_sampler.py

```diff
@@ -180,6 +182,19 @@ def __init__(self, failover=False, retry_interval=-1, **config):
         self.client = Client.from_config(**config)
         self.solver = self._get_solver(penalty=self._solver_penalty)
 
+    def close(self):
```

Comment:
Needs to be added to [here](https://github.com/dwavesystems/dwave-system/blob/1bb84c6e73513c66f31dc4bee06aeab6b9e6743d/docs/reference/samplers.rst?plain=1#L56)